src/lib/model/OutboundThirdpartyTransactionModel.js

```diff
@@ -0,0 +1,23 @@
+/**************************************************************************
+ *  (C) Copyright ModusBox Inc. 2020 - All rights reserved.               *
+ *                                                                        *
+ *  This file is made available under the terms of the license agreement  *
+ *  specified in the corresponding source code repository.                *
+ *                                                                        *
+ *  ORIGINAL AUTHOR:                                                      *
+ *       Sridhar Voruganti - sridhar.voruganti@modusbox.com               *
+ **************************************************************************/
+
+'use strict';
+
```

Comment:
I have asked to refactor the publication to channel code, so maybe putting it in this channel is a good idea.

Reply:
Ok